process/coordinator/process.go

```diff
@@ -703,6 +706,8 @@ func (tc *transactionCoordinator) CreateBlockStarted() {
 		value.CreateBlockStarted()
 	}
 	tc.mutInterimProcessors.RUnlock()
+
+	tc.transactionsLogProcessor.Clean()
```

Comment:
In this file at post processing - VerifyCreatedBlockTransactions you have the checks for receipts verification. the same should happen for log processor as well. hashes must match. Plus add an activation flag for that verification.

Reply:
added a TODO and Jira task